kustomizations/apps/data-hub-configs/ga-data-pipeline.config.yaml

```diff
@@ -7,7 +7,251 @@ importedTimestampFieldName: 'imported_timestamp'
 
 
 googleAnalyticsPipelines:
-  - pipelineID: 'ga_universal_analytics_data_1'
+  # - pipelineID: 'ga_universal_analytics_data_1'
```

Comment:
Do we need this commented out part?

Reply:
Yes, I wasn't sure. I thought maybe it could be useful to see what we did run before and is still in BigQuery for staging. But could also see that from history.
Happy to remove.